content/Rust-1.88.0.md

```diff
@@ -0,0 +1,109 @@
+
+```rust
+    fn visit_generic_args(&mut self, args: &'a ast::GenericArgs) {
+        if let ast::GenericArgs::Parenthesized(generic_args) = args
+            && let ast::FnRetTy::Ty(ref ty) = generic_args.output
+            && matches!(ty.kind, ast::TyKind::Never)
+        {
+            gate!(&self, never_type, ty.span, "the `!` type is experimental");
+        }
+        visit::walk_generic_args(self, args);
+    }
+```
+
+Let chains are only available in the Rust 2024 edition, as this feature depends on the [`if let` temporary scope](https://doc.rust-lang.org/edition-guide/rust-2024/temporary-if-let-scope.html) change for more consistent drop order. Earlier efforts tried to work with all editions, but some difficult edges cases threatened the integrity of the implementation. 2024 made it feasible, so please upgrade your crate's edition if you'd like to use this feature!
+
+### Naked functions
+
+Rust now supports writing naked functions with no compiler-generated epilogue and prologue, allowing full control over the generated assembly for a particular function. This is a more ergonomic alternative to defining functions in a `global_asm!` block. A naked function is marked with the `#[unsafe(naked)]` attribute, and its body consists of a single `naked_asm!` call, for example:
+
+```rust
+#[unsafe(naked)]
+pub unsafe extern "sysv64" fn wrapping_add(a: u64, b: u64) {
+    // Equivalent to `a.wrapping_add(b)`.
+    core::arch::naked_asm!(
+        "add rax, rdi, rsi",
+        "ret"
+    );
+}
+```
+
+The handwritten assembly block defines the _entire_ function body: unlike non-naked functions, the compiler does not add any special handling for arguments or return values. Naked functions are used in low-level settings like Rust's [`compiler-builtins`](https://github.com/rust-lang/compiler-builtins), operating systems, and embedded applications.
+
+Look for a more detailed post soon on [Inside Rust](https://blog.rust-lang.org/inside-rust/)!
+
+### Boolean configuration
+
+The `cfg` predicate language now supports boolean literals, `true` and `false`, acting as a configuration that is always enabled or disabled, respectively. This works in Rust [conditional compilation](https://doc.rust-lang.org/reference/conditional-compilation.html) with `cfg` and `cfg_attr` attributes and the built-in `cfg!` macro, and also in Cargo `[target]` tables in both [configuration](https://doc.rust-lang.org/cargo/reference/config.html#target) and [manifests](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies).
+
+If you are familiar with set theory, you might already be using empty sets like `cfg(all())` for enabled and `cfg(any())` for disabled, but this meaning is rather implicit and easy to get backwards. The boolean literals offer a more direct way to say what you mean.
```

Comment:
```suggestion
If you are familiar with set theory, you might already be using empty sets like `cfg(all())` for enabled and `cfg(any())` for disabled, but this meaning is rather implicit and easy to get backwards. `cfg(true)` and `cfg(false)` offer a more direct way to say what you mean.
```

Just adding some verbatim usage

```md
If you are familiar with set theory,
```

People not familiar with set theory aren't also using `cfg(all())`/`cfg(any())`? 😆

Reply:
Alternatively, just simply say sth like:

> Previously, users would use `cfg(any())` to emulate `cfg(false)`, and `cfg(all())` to emulate `cfg(true)`. Now, the user can use the more straightfoward `cfg(true)`/`cfg(false)` forms.